pages/user/[userId]/chat/[chatRoomId]/index.tsx

```diff
@@ -0,0 +1,90 @@
+import { Flex, Input, Text } from '@chakra-ui/react';
+import {
+  GetServerSideProps,
+  InferGetServerSidePropsType,
+  NextPage,
+} from 'next';
+import { useEffect, useState } from 'react';
+
+import { userAPI } from 'apis';
+import useStomp from 'hooks/useStomp';
+import { ChatMeesageResponseType } from 'types/chatMessages';
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
+  const { userId, chatRoomId } = context.query;
```

Comment:
```suggestion
export const getServerSideProps: GetServerSideProps = async ({ query: { userId, chatRoomId } }) => {
```

해당 부분를 이와 같이 줄여도 괜찮을 것 같습니다!

Reply:
반영하겠습니다!